modules/link-in-bio/module.php

```diff
@@ -44,17 +45,37 @@ public function __construct() {
 	/**
 	 * Register styles.
 	 *
-	 * At build time, Elementor compiles `/modules/link-in-bio/assets/scss/frontend.scss`
-	 * to `/assets/css/widget-link-in-bio.min.css`.
+	 * At build time, Elementor compiles `/modules/link-in-bio/assets/scss/widgets/*.scss`
+	 * to `/assets/css/widget-*.min.css`.
 	 *
 	 * @return void
 	 */
 	public function register_styles() {
```

Comment:
Can `register_styles()` go into the base module? Then each module only needs to declare it's `get_widget_style_list()`. WDYT?

Reply:
On it.

Reply:
Updated.